### PR TITLE
Updated the build process to update openshift source before build

### PR DESCRIPTION
**Pull request by mohammedzee1000**
This is for the dealing with dockerfiles not requiring tags in FROM stanza.

### CODE REVIEW

File: server/run.sh [hunk 1]
```diff
@@ -53,6 +53,12 @@ _ "Put build,test, delivery scripts in proper place"
 #echo "ADD /test_script /usr/bin/" >> $TARGET_FILE
 #echo "ADD /delivery_script /usr/bin/" >> $TARGET_FILE
 
+# Alter target file, if openshift container is detected
+CONTAINER_NAME="${APPID}/${JOBID}";
```
Comment (navidshaikh) [inline server/run.sh]: +1 on checking for matching the resulting container name

**Comment by bamachrn** [comment]
LGTM 👍

**Comment by bamachrn** [comment]
#dotests

**Comment by navidshaikh** [comment]
#dotests

**Comment by navidshaikh** [comment]
#dotests-debug

File: server/run.sh [hunk 2]
```diff
@@ -53,6 +53,15 @@ _ "Put build,test, delivery scripts in proper place"
...
+if [[ ${CONTAINER_NAME} =~ ^openshift\/(origin.*|node|openvswitch) ]]; then
+    _ "Openshift container detected, checking if  ${TARGET_FILE} needs to be altered";
+    if [[ ( ${DESIRED_TAG} == "v1.5" && ${CONTAINER_NAME} != "openshift/origin-base" ) && ${CONTAINER_NAME} != "openshift/origin-source" ]]; then
```
Comment (bamachrn) [inline server/run.sh]: why are we hardcoding to 1.5 here. cant we have any <desired_tag>. and exception for `latest` only?

Comment (mohammedzee1000) [inline server/run.sh]: @bamachrn we are only doing 1.5 and above and only 1.5 has the origin-base coming from centos 3.6 onwards, its origin-source

**Pull request merged**